app/bundles/PluginBundle/Config/config.php

```diff
@@ -118,6 +118,10 @@
                 'class'     => 'Mautic\PluginBundle\Helper\IntegrationHelper',
                 'arguments' => 'mautic.factory',
             ],
+            'mautic.helper.plugin.builder' => [
+                'class'     => 'Mautic\PluginBundle\Helper\BuilderHelper',
+                'arguments' => 'mautic.factory',
```

Comment:
Thanks. I've got that one fixed up, but then started digging myself a hole trying to remove `$this->factory` from all the other files touched by this. I'll keep at it because it'll be much easier to maintain for others in the future if I figure it out, but I can't guarantee I'll be able to manage it.